docs/concepts/otlp/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: OTLP
+sidebar_order: 400
+description: "Learn how to send OpenTelemetry trace data directly to Sentry from OpenTelemetry SDKs."
+keywords: ["otlp", "otel", "opentelemetry"]
+---
+
+<Include name="feature-available-for-user-group-limited-avail.mdx" />
+
+Sentry can ingest [OpenTelemetry](https://opentelemetry.io) traces directly via the  [OpenTelemetry Protocol](https://opentelemetry.io/docs/specs/otel/protocol/). If you have existing OpenTelemetry trace instrumentation, you can configure your OpenTelemetry exporter to send traces to Sentry directly.
+
+The easiest way to configure an OpenTelemetry exporter is with environment variables. You'll need to configure the trace endpoint URL, as well as the authentication headers. Set these variables on the server where your application is running.
+
+```bash {filename: .env}
+export OTEL_EXPORTER_OTLP_TRACES_ENDPOINT="___OTLP_TRACES_URL___""
+export OTEL_EXPORTER_OTLP_TRACES_HEADERS="x-sentry-auth=sentry sentry_key=___PUBLIC_KEY___"
+```
+
+Alternatively, you can configure the OpenTelemetry Exporter directly in your application code.
+
+```typescript {filename: app.ts}
+import { NodeSDK } from "@opentelemetry/sdk-node";
+import { OTLPTraceExporter } from "@opentelemetry/exporter-trace-otlp-http";
+
+const sdk = new NodeSDK({
+  traceExporter: new OTLPTraceExporter({
+    url: "___OTLP_TRACES_URL___",
+    headers: {
+      "x-sentry-auth": "sentry sentry_key=___PUBLIC_KEY___",
+    },
+  }),
+});
+
+sdk.start();
+```
```

Comment:
it might be prudent to add a note to https://docs.sentry.io/platforms/javascript/guides/node/opentelemetry/custom-setup/ here (indicating that if you are using OTEL yourself, you'll have to adjust your sentry sdk config).

Reply:
@AbhiPrasad sorry I'm not quite following, which document should link to which one and which one has the note?

Reply:
I was thinking we should link to https://docs.sentry.io/platforms/javascript/guides/node/opentelemetry/custom-setup/ in the case where a user runs a sentry sdk for errors-only, but then uses OTLP to send traces. The note would basically link to the `custom-setup` page indicating users should follow the guide written there to adjust their sentry sdk config accordingly.

Maybe this is overkill though. I'd also be open to note include this and just wait until we get user feedback.

Reply:
Not overkill IMO, added a note about this 👍🏻